src/components/ResourceList/components/FilterControl/FilterControl.tsx

```diff
@@ -44,11 +44,15 @@ export class FilterControl extends React.Component<CombinedProps> {
       polaris: {intl},
       context: {selectMode, resourceName},
     } = this.props;
+    const filterResourceName = resourceName || {
```

Comment:
The default was originally initialized in context. These changes move it to the render method